server/controllers/finance/cost_center_allocation_registry.js

```diff
@@ -0,0 +1,116 @@
+/**
+* Cost Center Allocation Keys
+*
+* This controller exposes an API to the client for reading allocation keys
+*/
+// const _ = require('lodash');
+const db = require('../../lib/db');
+const Stepdown = require('../../lib/stepdown');
```

Comment:
Normally, capital letters indicate a class name that you will invoke with `new`.  Would it be possible to import this as just `stepdown`?